src/Components/Logo/style.scss

```diff
@@ -0,0 +1,10 @@
+.logo-wrap {
```

Comment:
In scss you could use cascading logic
`
.logo {
  &-wrap {}
  &-left {}
  &-right {}
}
`

Reply:
Thanks for tip. 
The one possible disadvantage of using '&' is difficulties in 'search in files' in your editor. 
For example when you search 'logo-wrap' there would be no results.